app/main.py

```diff
@@ -2,0 +1,27 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    if command[0] == "mv" and len(command) == 3:
+        try:
+            mv, file1_name, destination = command
+            if destination.endswith("/"):
+                raise ValueError("Enter correct path and file name!")
+            dirs = destination.split("/")[:-1]
+            file2_name = destination.split("/")[-1]
+
+            dir_path = ""
+            for folder in dirs:
+                dir_path = os.path.join(dir_path, folder)
+                if not os.path.exists(dir_path):
+                    os.mkdir(dir_path)
```

Comment:
The current implementation does not account for existing directories, which can lead to errors. You should check if the directory exists before attempting to create it, as mentioned in the checklist .